gateway/assets/index.html

```diff
@@ -158,8 +158,8 @@ <h1 style="flex: 1 1 auto;">&nbsp; OpenFaaS Portal</h1>
                                 <md-input-container class="md-block" flex-gt-sm>
                                     <md-radio-group ng-model="invocation.contentType" layout="row" layout-align="start center">
                                         <md-radio-button value="text" class="md-primary"> Text </md-radio-button>
-                                        <md-radio-button value="json"> JSON </md-radio-button>
-                                        <md-radio-button value="binary"> Download </md-radio-button>                                        
+                                        <md-radio-button value="json" class="md-primary"> JSON </md-radio-button>
+                                        <md-radio-button value="binary" class="md-primary"> Download </md-radio-button>                                        
```

Comment:
I also fixed the radio button colors of JSON and Download. Somehow they were pink while the Text radio button is Blueish.

@alexellis Do you know if this is intended? If it is, I'll revert it.

Reply:
I'm not sure.

Reply:
**Before**
<img width="303" alt="ss 2018-08-23 22 07 20" src="https://user-images.githubusercontent.com/3941889/44527163-0685ee00-a721-11e8-8412-cb5267c56d6b.png">

**After**
<img width="299" alt="ss 2018-08-23 22 06 30" src="https://user-images.githubusercontent.com/3941889/44527182-10a7ec80-a721-11e8-8902-4142767fe134.png">

Reply:
I think it's natural to have them all with the same color (blueish, purple)

Reply:
Agreed.